src/tests/intro.md

```diff
@@ -170,7 +170,38 @@ communicate with the server to coordinate running tests (see
 
 ## Crater
 
-TODO
+[Crater](https://github.com/rust-lang-nursery/crater) is a tool for compiling
+and running tests for _every_ crate on [crates.io](https://crates.io/). It is
+mainly used for checking for extent of breakage when implementing potentially
+breaking changes.
+
+### When to run Crater
+
+You should request a crater run if your PR makes large changes to the compiler
+or could cause breakage. If you are unsure, feel free to ask your PR's reviewer.
+
+### Requesting Crater Runs
+
+The rust team maintains a few machines that can be used for running crater runs
+on the changes introduced by a PR. If your PR needs a crater run, leave a
+comment for the triage team in the PR thread. Your will be enqueued by the
+triage team and the results will be posted when they are ready. A crater run
+usually takes a few days (as of this writing).
```

Comment:
Please inform the team whether you require a "check-only" crater run, a "build only" crater run, or a "build-and-test" crater run. The difference is primarily in time; the conservative (if you're not sure) option is to go for the build-and-test run. If making changes that will only have an effect at compile-time (e.g., implementing a new trait) then you only need a check run.

Check runs will take around ~3-4 days, with the other two taking 5-6 days on average.